src/pepr/operator/controllers/network/policies.ts

```diff
@@ -86,9 +98,15 @@ export async function networkPolicies(pkg: UDSPackage, namespace: string, istioM
   const ssos = pkg.spec?.sso?.filter(sso => sso.enableAuthserviceSelector);
 
   for (const sso of ssos || []) {
+    // In ambient mode, target the waypoint pods instead of the app pods if the package should use ambient waypoint
+    const netpolSelector =
+      istioMode === IstioState.Ambient && shouldUseAmbientWaypoint(pkg)
```

Comment:
nit: isn't checkin istioMode here redudant since `shouldUseAmbientWaypoint(pkg)` does that too?
```suggestion
      shouldUseAmbientWaypoint(pkg)
```

Reply:
good catch i missed this in my refactor